### PR TITLE
Add unit tests for Bean validation

### PR DESCRIPTION
**Pull request by kj84park**
# What is this PR for?

## Overview or reasons
- Added unit tests related to the issues below
#20

### CODE REVIEW

File: server/src/test/java/com/linecorp/line/auth/fido/fido2/server/bean/validation/mockmvc/AuthChallengeRequestBeanValidationTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    @Test
+    void validateSuccessfulRequest() throws Exception {
+
+        mockMvc.perform(post(AUTH_CHALLENGE_URL_PATH)
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(objectMapper.writeValueAsString(authChallengeRequest)))
+                .andExpect(status().isBadRequest());
+    }
```
Comment (seachicken) [inline server/src/test/java/com/linecorp/line/auth/fido/fido2/server/bean/validation/mockmvc/AuthChallengeRequestBeanValidationTest.java:48]: If the production code and the test code are packaged differently, we may not notice that there are tests.
How about writing them in `ChallengeControllerTest` as the same web layer test?

**Review by seachicken — COMMENTED** [review]
Good job! I'll be relieved when the test is added.

File: server/src/test/java/com/linecorp/line/auth/fido/fido2/server/bean/validation/mockmvc/AuthChallengeRequestBeanValidationTest.java (hunk 1 above)
Comment (kj84park) [reply server/src/test/java/com/linecorp/line/auth/fido/fido2/server/bean/validation/mockmvc/AuthChallengeRequestBeanValidationTest.java]: Thank you for taking the time to give us your suggestions! As you mentioned, I've made modifications!

**Comment by kj84park** [comment]
Thank you for taking the time to give us your suggestions! As you mentioned, I've made modifications!

File: common/src/test/java/com/linecorp/line/auth/fido/fido2/common/server/bean/validation/AuthOptionRequestBeanValidationTest.java [hunk 2]
```diff
@@ -40,9 +40,10 @@ void validateIncompleteRequestWithBlank() {
 
         final Set<ConstraintViolation<AuthOptionRequest>> constraintViolations = validator.validate(authOptionRequest);
 
-        assertThat(constraintViolations.size()).isEqualTo(1);
+        assertThat(constraintViolations).hasSize(1);
         constraintViolations.forEach(
```
Comment (seachicken) [inline common/src/test/java/com/linecorp/line/auth/fido/fido2/common/server/bean/validation/AuthOptionRequestBeanValidationTest.java:44]: [containsOnly](https://www.javadoc.io/doc/org.assertj/assertj-core/latest/org/assertj/core/api/AbstractIterableAssert.html#containsOnly(ELEMENT...)) checks all the elements of a collection, so we can get rid of `forEach`

Comment (kj84park) [reply common/src/test/java/com/linecorp/line/auth/fido/fido2/common/server/bean/validation/AuthOptionRequestBeanValidationTest.java]: Oh, thanks!

**Review by seachicken — APPROVED** [review]
Good work 👍

**Pull request merged**